docs/src/user-manual/overview.md

```diff
@@ -0,0 +1,100 @@
+# Overview and getting started with Enduro
+
+[Enduro][Enduro] is a new application under development by [Artefactual
+Systems]. Designed following digital preservation standards and best practices,
+our goal with Enduro is to ensure records of the past can be cared for in the
+present and trusted in the future. The application borrows terminology from the
+digital preservation domain - a full glossary of terms and how we define them in
+the Enduro project can be found at:
+
+* [Glossary](glossary.md)
+
+Originally created as a more stable replacement for Archivematica’s
+[automation-tools][automation-tools] library of scripts, Enduro has since
+evolved into a flexible tool to be paired with preservation applications to
+provide initial ingest activities such as content and structure validation,
+packaging, and more. To learn more about the components that make up Enduro,
+see:
+
+* [Components](components.md)
+
+Below you'll find basic instructions on how to login to the application and an
+overview of some of the first pages you will encounter. While the Enduro user
+interface provides a single seamless experience for operators, behind the scenes
+the code is organized into functional modules based on different stages of the
+digital preservation lifecycle. Currently these include:
+
+* [Ingest](ingest/index.md)
+* Storage
```

Comment:
It maybe feels redundant to have this paragraph when the sidebar nav is pretty prominent? Not a big deal though.

Reply:
I'm gonna leave this one - I get that there's sidebar nav, but this is also kind of a "this is how we have organized the docs and why section. And, sometimes a little redundancy in nav options can be helpful.